mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -108,6 +108,8 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
       totalUSDValue: new BigNumber(0),
       dstTokens: []
     })
+    // to sort USD values in descending order
+    const sortedUsdAmt = dstTokens.sort((a, b) => parseInt(b.usdAmount.toFixed(8)) - parseInt(a.usdAmount.toFixed(8)))
```

Comment:
![image](https://user-images.githubusercontent.com/13292662/153350292-382d882f-31f8-4795-bb75-8bdbf8e9b539.png)

`parseInt` would remove the decimal points, could you check on how you can sort it with BigNumber funcs?